AdyenActions/UI/View Controllers/QR Code/QRCodeView.swift

```diff
@@ -55,7 +55,7 @@ internal final class QRCodeView: UIView, Localizable, Observer {
         logo.translatesAutoresizingMaskIntoConstraints = false
         NSLayoutConstraint.activate([
             logo.bottomAnchor.constraint(equalTo: topAnchor, constant: -7),
-            logo.centerXAnchor.constraint(equalTo: centerXAnchor)
+            logo.centerXAnchor.constraint(equalTo: safeAreaLayoutGuide.centerXAnchor)
```

Comment:
Somehow I belive that centerX is always center?

Reply:
i think if there is a view that has safe area on the left, but no safe area on the right (does not span the full width of the phone) - `centerX` might be different.

Reply:
Than center of proggress bar and center of instructionLabel would be different 🤔
I would say better align those two